### PR TITLE
WIP: employee onboarding process

### PR DESCRIPTION
**Pull request by stickgrinder**
@paolomainardi @aysemeric I'd ask some help to review this page and see how it can be improved.
I surely forgot something important.

There are things we are not currently doing, I plan to structure all the information to do them and try to make them more automatic possible!

### CODE REVIEW

File: content/recipes/employee-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* Add the account to Slack and _invite_ it on `#general` and `#tech` as well as on all channels related to relevant projects
+
+## Things to do on the first day
+
+### For the company
+
+* HR must introduce the new employee to the resident team and, on slack `#general` channel, to the remote colleagues
+* HR must assign a buddy (usually her team leader) to the new employee
+* Tech must make sure the local development environment is working on the employee's PC
+* HR must send an introductory e-mail with policies (links) about
```
Comment (aysemeric) [inline content/recipes/employee-onboarding.md]: an introduction our Google suite like Drive and Calender and how to use it

File: content/recipes/employee-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+* Open the Gitlab account and add it to all relevant projects
+* Open the Toggl account and make sure it can access all relevant projects
+* Add the account to Slack and _invite_ it on `#general` and `#tech` as well as on all channels related to relevant projects
+
+## Things to do on the first day
+
+### For the company
+
+* HR must introduce the new employee to the resident team and, on slack `#general` channel, to the remote colleagues
+* HR must assign a buddy (usually her team leader) to the new employee
```
[the remark this answers is not in the document]
Comment (stickgrinder) [reply content/recipes/employee-onboarding.md]: To date, feminine form is the accepted neutral gender on official English writings.

File: content/recipes/employee-onboarding.md (hunk 1 above)
Comment (stickgrinder) [reply content/recipes/employee-onboarding.md]: @aysemeric I added this below:

```
  * Google Drive folder structure and usage
  * Google Calender configuration and conventions
```

Comment (stickgrinder) [reply content/recipes/employee-onboarding.md]: If you feel it's ok, resolve this conv.

File: content/recipes/employee-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+* Tech must make sure the local development environment is working on the employee's PC
+* HR must send an introductory e-mail with policies (links) about
+  * Resource access
+  * Time tracking
+  * Management of off/sick days
+  * How and whom to ask for help
+  * 1:1 meetings and performance review policies (ISCs)
+* Tech should send in introductory e-mail with information (links) about
```
[the remark this answers is not in the document]
Comment (stickgrinder) [reply content/recipes/employee-onboarding.md]: Added below as a key point:

```
* Tech and/or buddy must explain our key internal projects and self-made tools (sparkdrupal, sparkangular, sparkboard, firestarter, etc) to the employee
```

**Pull request merged**